src/validators/number/number.validator.md

```diff
@@ -25,4 +26,5 @@ const result = validateSchema(schema, value)
 | min | number | no | | validate if the number is equal or greater then the min |
 | max | number | no | | validate if the number is equal or less then the max |
 | positive | boolean | no | false | validate if the number is positive (0 is considered positive) |
-| negative | boolean | no | false | validate if the number is negative |
\ No newline at end of file
+| negative | boolean | no | false | validate if the number is negative |
+| strict | boolean | no | true | if false, it will try to convert 
```

Comment:
little improvement here:

`if false, will try to convert the value using new Number(value) |` (don't forget the `|`) at the end.